lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,445 @@
+    chaser=['alembic'],
+    chaserArgs=[
+       ('alembic', 'primvarprefix', 'ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:'),
+    ])
+```
+
+
+## Setting Site-Specific Defaults for MayaUSDImportCommand/MayaUSDExportCommand
+
+Suppose that at your site you always want to export with the flags
+`-exportMaterialCollections` and `-chaser alembic` and `-chaserArgs ...`,
+even when exporting using the "File > Export All" menu item, where
+you wouldn't normally be able to set some more advanced options like
+`-chaser` that are only available via the specific command, for example
+the `mayaUSDExport` command.
+
+You can configure these options as the "site-specific" defaults for
+your installation of the plug-in, for example the Autodesk Maya USD
+plugins by creating a plugin and adding some information to its
+`plugInfo.json` file (see the USD Plug library documentation for more
+information).
+
+For example, your `plugInfo.json` would contain these keys if you
+wanted the default flags mentioned above:
+
+```javascript
+{
+    "UsdMaya": {
+        "mayaUSDExport": {
+            "exportMaterialCollections": true,
+            "chaser": "alembic",
+            "chaserArgs": [
+                ["alembic", "primvarprefix", "ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:"],
+                ["alembic", "attrprefix", "ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:"]
+            ]
+        }
+    }
+}
+```
+
+This also works for the `MayaUSDImportCommand` base command, for example
+in the `mayaUSDImport` command and the "File > Import" menu item; use
+the `mayaUSDImport` key in the `plugInfo.json` file to configure your
+site-specific defaults.
+
+
+## `mayaUSDListShadingModesCommand`
+
+The purpose of this command is to translate between nice-names, internal
+names and annotations for various elements passed to the other commands.
+
+### Command Flags
+
+| Long flag              | Short flag | Type           | Description |
+| ---------------------- | ---------- | -------------- | ----------- |
+| `-export`              | `-ex`      | noarg          | Retrieve the list of export shading mode nice names. |
+| `-exportOptions`       | `-eo`      | string         | Retrieve the real name of the shading mode given its nice name. These can be used in the `shadingMode` option of the export command. |
+| `-exportAnnotation`    | `-ea`      | string         | Retrieve the description of the export shading mode option |
+| `-findExportName`      | `-fen`     | string         | Retrieve the nice name of an export shading mode |
+| `-findImportName`      | `-fin`     | string         | Retrieve the nice name of an import shading mode |
+| `-import`              | `-im`      | noarg          | Retrieve the list of import shading mode nice names. |
+| `-importOptions`       | `-io`      | string         | Retrieve the real name of the shading mode given its nice name. These can be used in the `shadingMode` option of the export command. |
```

Comment:
Returns a pair of strings that completely defines a `shadingMode` entry, as used by the USD import options.

Reply:
done

Reply:
The review comment was for the line above. This means the fix you did to "-importAnnotation" should have been done to "-importOptions" instead.

Reply:
Done.